src/components/MarketSelector/PoolListItem.tsx

```diff
@@ -44,6 +45,7 @@ export function PoolListItem(props: {
     showBalances,
     onFavoriteClick,
     onSelectOption,
+    chainId,
```

Comment:
I think it's better to take it from selector, or it's not possible?

Reply:
Possible, used selector